samcli/commands/local/cli_common/user_exceptions.py

```diff
@@ -10,68 +10,50 @@ class InvokeContextException(UserException):
     Something went wrong invoking the function.
     """
 
-    pass
-
 
 class InvalidSamTemplateException(UserException):
     """
     The template provided was invalid and not able to transform into a Standard CloudFormation Template
     """
 
-    pass
-
 
 class SamTemplateNotFoundException(UserException):
     """
     The SAM Template provided could not be found
     """
 
-    pass
-
 
 class DebugContextException(UserException):
     """
     Something went wrong when creating the DebugContext
     """
 
-    pass
-
 
 class ImageBuildException(UserException):
     """
     Image failed to build
     """
 
-    pass
-
 
 class CredentialsRequired(UserException):
     """
     Credentials were not given when Required
     """
 
-    pass
-
 
 class ResourceNotFound(UserException):
     """
     The Resource requested was not found
     """
 
-    pass
-
 
 class InvalidLayerVersionArn(UserException):
     """
     The LayerVersion Arn given in the template is Invalid
     """
 
-    pass
-
 
 class UnsupportedIntrinsic(UserException):
```

Comment:
We dont need a pass anymore? Didnt know this. Good to know.

Reply:
https://github.com/PyCQA/pylint/issues/2208